views/js/runner/plugins/content/responsiveness/collapser.js

```diff
@@ -37,12 +37,29 @@ define([
     var noLabelHoverCls = 'no-tool-label-hover';
 
     /**
+     * Allow to set manually which buttons should collapse and in which order.
+     * This can be set by triggering the "collapser-set-order" event on the testRunner.
+     * Given as an array of jQuery selectors: first index will be the first to be collapsed, and so on.
+     * If no selector is given for a button, then this one will never collapse.
+     * ex:
+     * var collapseOrder = [
+     *      '[data-control="highlight-clear"],[data-control="highlight-trigger"]',  // those will collapse first...
+     *      '[data-control="hide-review"]',                                         // this one second...
+     *      '[data-control="set-item-flag"]',                                       // third...
+     *      ...                                                                     // ...
+     * ];
+     * @type {String[]}
+     */
+    var collapseOrder = [];
```

Comment:
Why are you using an event instead of a config entry? Is there any chance the list may change at runtime? In the companion PR the list is set roughly at init. So that seems to be artificial, isn't it? Or is there something I didn't see/understand?